### DISCUSSION TITLE
Still under Development??

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Voidware-Prohibited/ALSXT#37] [General]
Is this still being worked on??

**Comment by smateja** [comment]
I hope so also. The Project is looking good. Hope they update this. @Devs => If this is a motivation Problem - feel free to contact me - maybe i can help an place Donations etc.

**Reply by colorindarkness** [reply]
Yes, I just re-located, so anything helps!

**Comment by colorindarkness** [comment]
Hey, everyone. I was on hiatus while I re-located to California. Development will resume shortly!

**Comment by smateja** [comment]
Sounds Good and welcome back. Can we help maybe with a donation or Else?
________________________________

**Reply by colorindarkness** [reply]
Yes. PayPal: @colorindarkness . I will setup a Sponsorship account soon! Thank you!

**Comment by smateja** [comment]
Is it possible that you open a Patreon site - i will Look forward to collect Patreons as well...
________________________________

**Reply by colorindarkness** [reply]
Ok. I have setup a Patreon! Thank you! [patreon.com/colorindarkness](https://www.patreon.com/colorindarkness)

**Comment by smateja** [comment]
👍 🤙
________________________________
